drv/stm32h7-sprot-server/src/main.rs

```diff
@@ -66,7 +70,7 @@ const TIMEOUT_WRITE_ONE_BLOCK: u32 = 2_000;
 const PART1_DELAY: u64 = 0;
```

Comment:
PART1_DELAY should be non-zero to work around issue 929 at the cost of some performance.

Reply:
...but that should be carefully tested and in a separate PR.